api/v2/openapi.yaml

```diff
@@ -523,12 +523,12 @@ definitions:
         type: string
       sendResolved:
         type: boolean
-      lastNotify:
+      lastAttempt:
```

Comment:
```suggestion
      lastAttempt:
          description: a timestamp of last attempt to use the integration
```

Reply:
I thought of adding descriptions this way but my concern is that no other structs have a description in the spec. Do you think we should do it anyway?

Reply:
Btw I totally agree that an actual description would be better than the auto-generated one, I'm just not sure wether we should do it now for this particular struct.

Reply:
I think more comments are better.

Reply:
Ok, I'll add a next commit with the spec updated and the needed files generated.